src/payment/strategies/googlepay/googlepay-payment-processor.ts

```diff
@@ -197,4 +208,36 @@ export default class GooglePayPaymentProcessor {
             customFields: [],
         };
     }
+
+    private _postForm(postPaymentData: TokenizePayload): Promise<Response<any>> {
```

Comment:
Do you really want to return `any`? If the response is empty, I think `Promise<Response<void>>` might be more appropriate.